lib/Util.php

```diff
@@ -29,4 +29,12 @@ public static function getStunServer(IConfig $config) {
         return $config->getAppValue('spreed', 'stun_server', 'stun.l.google.com:19302');
     }
 
+    public static function getTurnSettings(IConfig $config, string $uid) {
```

Comment:
Seems kinda strange to always have to inject IConfig and so on. Maybe make it non-static? :-)